### ISSUE TITLE
File system can't be found in Ubuntu 19.04

### ISSUE THREAD

**Issue by vishalds2003** [jeffThompson/DarkArduinoTheme#28]
In Ubuntu 19.04 Disco Dingo, this code can't be used as the file tree specified in the README.md is incorrect. I don't know if this is a problem that persists only in Ubuntu 19.04 (specially as this is a newly released Ubuntu distro) but there is really no folder named arduino in the /usr/share folder.

When I give the `ls` command in the /usr/share directory, all the the directories beginning with the letter a are just 
`aclocal`
`acpi-support`
`alsa`
`alsabase`
`appdata`
`appinfo`
`app-install`
`applications`
`apport`
`apps`
`appturl`
`appspell`
`avahi`

I have searched for folders named arduino system-wide using **Grep**. But still, to no gain. There are no folders namely Arduino at all throughout the system except for the library folders..

**Comment by jeffThompson** [comment]
Hmm, I don't have an Ubuntu install to test on. You could try a search for `theme.txt` to find the original theme file? Or maybe a `which arduino` search?

**Comment by vishalds2003** [comment]
OK. I've found out the folder at last. So now, here's the update : If you're running Ubuntu 19.04 (I don't have any other versions and so can't really comment on those) and have installed the Arduino IDE directly from the [official downloads page](https://www.arduino.cc/en/Main/Software) (not from Ubuntu Software), then it installs using opt. And hence, the file system in this case would be `/opt/arduino-1.8.9-linux64/arduino-1.8.9/lib/theme`. The names of the arduino folders may vary depending on your IDE version. But, the file tree will remain the same. 
Just replace the theme folder with the one found in this repo and bingo, you've got your new dark theme running. 
By the way, great work @jeffThompson . Well done and thanks for your repo...

**Comment by vishalds2003** [comment]
@jeffThompson please do update the [README.md](https://github.com/jeffThompson/DarkArduinoTheme/blob/master/README.md) accordingly so that everyone who comes here won't have to bother about this hereafter...

**Comment by jeffThompson** [comment]
Thanks for this but I'm not sure I quite follow. If you don't do the direct install, does it not work? Or was it in another location?

**Comment by vishalds2003** [comment]
I remember installing this in Ubuntu 18.04 from the Ubuntu Software. Then, it was in another location... I don't clearly remember. Also, I don't have an Ubuntu 18.04 install to test with..
So, not sure...

**Comment by jeffThompson** [comment]
Got it – you had to re-install the Arduino IDE to get it to work. Thanks again!

**Comment by vishalds2003** [comment]
I've just noticed that you still have given the file tree `/usr/share/arduino/lib`. I had already informed you that if you install directly from the official arduino page, then the tree will be `/opt/arduino-1.8.9-linux64/arduino-1.8.9/lib/theme`. I had installed mine from the arduino official page. Earlier, when I was running 18.04, I had installed the IDE from Ubuntu Software. That's what I told. I don't remember what the file tree was like back then. But, now, when we install from the official downloads page, it installs using opt. 

So, the README should be like:
- Mac users should look in ~/Applications/Arduino.app/Contents/Java/lib and replace the theme folder inside (making a copy of the original in case want to revert back).
- Windows is located in C:\Program Files (x86)\Arduino\lib
- Linux will be in `/usr/share/arduino/lib` - As for Ubuntu distros, it will be in `/opt/arduino-x.x.x-linux64/arduino-x.x.x/lib/theme` as Ubuntu installs via opt...

**Comment by vishalds2003** [comment]
I've just noticed that you still have given the file tree as `/usr/share/arduino/lib`. I had already informed you that if you install directly from the official arduino page, then the tree will be `/opt/arduino-1.8.9-linux64/arduino-1.8.9/lib/theme`. I had installed mine from the arduino official page. Earlier, when I was running 18.04, I had installed the IDE from Ubuntu Software. That's what I told. I don't remember what the file tree was like back then. But, now, when we install from the official downloads page, it installs using opt.

So, the README should be like:

- Mac users should look in `~/Applications/Arduino.app/Contents/Java/lib` and replace the theme folder inside (making a copy of the original in case want to revert back).
- Windows is located in `C:\Program Files (x86)\Arduino\lib`
- Linux will be in `/usr/share/arduino/lib` - As for Ubuntu distros, it will be in `/opt/arduino-x.x.x-linux64/arduino-x.x.x/lib/theme` as Ubuntu installs via opt...
